sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<p><code>matplotlib</code> flag  should be set to False. If it is set to True, then </p><p>it shows the following warning</p><pre class="ql-syntax" spellcheck="false">posx and posy should be finite values
</pre><p>and no plot is drawn. I speculate this is because some features are not numerical and matplotlib does not know how to handle it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Fixed</p>